migrations/20201125221909-create-event-participant.js

```diff
@@ -0,0 +1,40 @@
+"use strict";
+module.exports = {
```

Comment:
You can run few commands in one migration. But in such case you need to wrap them into transaction. This makes sense in cases, when you do so for one usecase. Here, for example, this could be done, cause this is like one operation 'db initialization'.